### PR TITLE
Add screenshots for each theme to client documentation

### PR DESCRIPTION
**Pull request by W7PEA**
New themes page and screenshots for each of them.

### CODE REVIEW

**Comment by MaxLeiter** [comment]
The images look good, but I don't see the themes page

**Comment by W7PEA** [comment]
OK... tested it with jekyll on my local box. Looks good to go to me.

**Comment by W7PEA** [comment]
crypto_css
example_css
morning_css
zenburn_css

**Comment by W7PEA** [comment]
themes_page

**Comment by MaxLeiter** [comment]
Looks great :+1:

File: _docs/client/themes.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: documentation
+title: Themes
+description: Add your description here
```
Comment (astorije) [inline _docs/client/themes.md]: Did you forget to type in the description?

File: _docs/client/themes.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+category: Client
+order: 3.3
+---
+
+# Custom Themes
+
+Custom themese can be be configured in the config.js file.
```
Comment (astorije) [inline _docs/client/themes.md]: Tpyo at "themes"

File: _docs/client/themes.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+```javascript
+//
+// Set the default theme.
+//
+// @type     string
+// @default  "themes/example.css"
+//
+theme: "themes/example.css",
+```
+The Lounge ships with 4 themes
```
Comment (astorije) [inline _docs/client/themes.md]: Missing period (I'm that nitpicky :-) )

**Comment by astorije** [comment]
@W7PEA, what a great addition!

I have a few comments though, to add to my inline comments:

- Can you erase the red file names in the screenshots themselves? Having their name in the section title is enough.
- The pictures are too big, could you size them smaller (maybe half?) with a link to the raw pictures?
- I know this has effect on the client only, but at the moment this is set at the server config, as your intro states. I think it would be more efficient to add the 4 images wrapped in `col-md-6` divs in http://thelounge.github.io/docs/server/configuration.html#theme, what do you think? When we improve how themes are handled it will be a different story but at the moment it seems more appropriate to me.

Thanks for this contrib!

File: _docs/client/themes.md (hunk 1 above)
Comment (astorije) [inline _docs/client/themes.md]: I see now that all pages have this... Urgh. I'd say we should start changing that but in a later comment I mention that I'd rather see this in the `theme` section of the configuration doc.

**Comment by W7PEA** [comment]
Sounds good. 

Do you have an example of wrapping divs around markdown in an .md? I just tried using markdown="1" on the div which is supposed to work, but the markdown in the div is not being converted to HTML. If not, i'll just slap it in with HTML.

**Comment by astorije** [comment]
@W7PEA, I couldn't make this work either (with things like `{ .col-md-6 }` and many declinations of it so I would say that you should go ahead and do HTML for these, unless someone who has already done that **for GitHub Pages** can help (I have already made that work in Markdown so that makes it frustrating, but that wasn't GitHub Pages nor Jekyll...).

Maybe we'll get more lucky when switching to Kramdown (as part of #3): http://kramdown.gettalong.org/syntax.html#attribute-list-definitions

Oh, last thing, when you're done make sure you squash your commits, that's a shady history you got there :p

**Comment by W7PEA** [comment]
Hi there, I was out of town for a couple of days... 
Using one narrower div was caussing other issues with the markdown layout and since you are planing to switch to kramdown in the future, I didn't want to make this page a completer one-off just for four screenshots. Instead, I used the img-responsive class in bootstrap to manage the size appropriately. 

capture


And I squashed to cover up my shady past. ;-)

**Comment by astorije** [comment]
Hi @W7PEA!

Do not take the transition to kramdown for granted. First of all, it might not happen before months, and if/when it does, chances are it will introduce no changes whatsoever.
Unless @JocelynDelalande comes up with a plan for restructuring docs, sets it in motion, and that plan involves getting rid of the screenshots (which I highly doubt re: getting rid of the screenshots), your work is going to stay so better get it right :-)

I am still not in favor of displaying super big screenshots. the 4 on a single line for big screens, 2 per line on small resolutions, clickable as you made them, this is more sensible IMO.

Do you think you could improve that? :-)

And thanks for squashing!

**Comment by W7PEA** [comment]
Hope this works for you.

**Comment by MaxLeiter** [comment]
Bump?

**Comment by astorije** [comment]
Woops, thumbnails for crypto and example are switched :-)

**Comment by W7PEA** [comment]
The simplest contribution ever with the most churn ever.

**Comment by astorije** [comment]
:+1: 
@JocelynDelalande?

**Comment by JocelynDelalande** [comment]
ok for me, thanks @W7PEA :)

:+1:

**Pull request merged**